src/popup/snooze-content.js

```diff
@@ -66,6 +72,7 @@ function openSnoozedTab(item) {
     active: true,
     url: item.url
   });
+  cancelSnoozedTab(item);
```

Comment:
I do wonder a little about this…  Was that part of the spec?  If not, let's run it by @sevaan to see what he thinks.